src/composables/useBadgeModal.js

```diff
@@ -30,6 +30,12 @@ export const ID_US_ECONOMIC_EQUALITY = 'us-economic-equality';
 export const ID_CLIMATE_ACTION = 'climate-action';
 export const ID_REFUGEE_EQUALITY = 'refugee-equality';
 export const ID_BASIC_NEEDS = 'basic-needs';
+// TODO: Use the LoanChannelQueryMap to get the correct filters
```

Comment:
Is a ticket logged for this?

Reply:
The query map will work if our achievements directly match existing channels in that map file. Do our achievements as defined match existing channels?